src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,14 @@
+<configuration scan="true">
+    <springProfile name="dev,prod">
+        <appender name="consoleJsonAppender" class="ch.qos.logback.core.ConsoleAppender">
+            <encoder class="net.logstash.logback.encoder.LogstashEncoder"/>
+        </appender>
+        <root level="INFO">
+            <appender-ref ref="consoleJsonAppender"/>
+        </root>
+    </springProfile>
+    <springProfile name="local">
+        <include resource="org/springframework/boot/logging/logback/base.xml"/>
+        <logger name="org.springframework.web" level="DEBUG"/>
```

Comment:
Er det ikke for mye logging med `debug` på spring-web?

Reply:
Det var "DEBUG" som var grunnen til at jeg la inn denne configgen egentlig, jeg ville se hva vi sendte til Altinn gitt requesten inn til oss. Men det er bare debug på local, i dev og prod ligger det på info. Samme som de andre applikasjonene vi har :)